frontend/src/components/eMIB/SideNavigation.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from "react";
+import { Tab, Row, Col, Nav } from "react-bootstrap";
+import { HEADER_HEIGHT, FOOTER_HEIGHT } from "../eMIB/constants";
+import PropTypes from "prop-types";
+
+const BODY_HEIGHT = `calc(100vh - ${HEADER_HEIGHT + FOOTER_HEIGHT}px)`;
```

Comment:
Should we put that const in _constants.js_, since the header and footer height are defined there?

Reply:
Since it has actual CSS in it, I'd rather leave it in each component, and the actual number values in constants.js. It also seems like it gets calculated differently in different components?